align_system/configs/adm/outlines_regression_aligned_comparative/incontext.yaml

```diff
@@ -7,11 +7,8 @@ inference_kwargs:
     number: 5
     method: prompt_bert_similarity
     leave_one_out: false
-    normalization: globalnorm
+    normalization: null
+    sort_actions: true
```

Comment:
I have some concern about changing default options without more fanfare, as someone could think they're re-running a previous config that we submitted for evaluation but be using these new settings instead without really noticing (something I probably would have done).  Does this actually change the `normalization` or is the default behavior (if `normalization: null` to use `globalnorm`)?

Reply:
When I added the option to normalize ICL scores I changed this default and shouldn't have. null is equivalent to rawscores or no normalization which was the default for DRE. I think it makes more sense to turn this off by default, I don't think we will end up using ICL norm in the final config

Reply:
Ah gotcha, good to know.  I don't have much objection here then (and I suspect that changing the `sort_actions` to be true by default doesn't make a tremendous impact on system behavior?).  When you update the changelog please note the change in the default behavior in the `Changed` section so there's a more noticeable record of the change